Mathlib/Topology/Algebra/Nonarchimedean/AdicTopology.lean

```diff
@@ -224,8 +224,8 @@ instance (priority := 100) : NonarchimedeanRing R :=
 instance (priority := 100) : UniformSpace R :=
   IsTopologicalAddGroup.toUniformSpace R
 
-instance (priority := 100) : UniformAddGroup R :=
-  uniformAddGroup_of_addCommGroup
+instance (priority := 100) : IsUniformAddGroup R :=
```

Comment:
Do we want a deprecation here?

Reply:
The name is autogenated, I don't see anything to deprecate.